src/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -2104,6 +2104,22 @@ public Notification createNotification(Song song, int state, int mode)
 		}
 		if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
 			notification.visibility = Notification.VISIBILITY_PUBLIC;
+
+			RemoteViews viewsPublic = new RemoteViews(getPackageName(), R.layout.notification);
+			viewsPublic.setInt(R.id.title, "setText", R.string.app_name);
+
+			viewsPublic.setImageViewResource(R.id.cover, R.drawable.icon);
+			viewsPublic.setImageViewResource(R.id.play_pause, playButton);
+			viewsPublic.setOnClickPendingIntent(R.id.play_pause, PendingIntent.getService(this, 0, playPause, 0));
+			viewsPublic.setOnClickPendingIntent(R.id.next, PendingIntent.getService(this, 0, next, 0));
+			viewsPublic.setOnClickPendingIntent(R.id.close, PendingIntent.getService(this, 0, close, 0));
+			viewsPublic.setViewVisibility(R.id.close, closeButtonVisibility);
+
+			Notification notificationPublic = notification.clone();
+			notificationPublic.contentView = viewsPublic;
+
+			notification.visibility = Notification.VISIBILITY_PRIVATE;
```

Comment:
VISIBILITY_PRIVATE is the default, so line 2106 can just be removed instead of overwriting it again.